pkg/driver_sx126x/contrib/driver_sx126x_hal.c

```diff
@@ -28,10 +28,35 @@
 
 #define ENABLE_DEBUG 0
 #include "debug.h"
+#include "cpu.h"
 
 #define SX126X_SPI_SPEED    (SPI_CLK_1MHZ)
 #define SX126X_SPI_MODE     (SPI_MODE_0)
 
+#ifndef SX126X_PERIPH_WAKEUP_TIME
+#if defined(CPU_FAM_STM32WL)
+#define SX126X_PERIPH_WAKEUP_TIME   10000U
+#else
+#define SX126X_PERIPH_WAKEUP_TIME   500U
+#endif
+#endif
+
+#if defined(CPU_FAM_STM32WL)
+static bool subghz_radio_sleepstatus = false;
```

Comment:
Shouldn't this be in the driver descriptor?

Reply:
`subghz_radio_sleepstatus` is currently only used by the SPI_HAL. The upper layer does not implement any  `sleep` state specific calls.

Reply:
The `NETOPT_STATE_SLEEP` command is still not implemented, but it should come soon. Therefore, I'm in with @fjmolinas that this should be in the device descriptor.

Reply:
Done.